tests/unit/test_response/test_serialization.py

```diff
@@ -1,71 +1,93 @@
 import enum
-from json import loads
 from pathlib import Path, PurePath
-from typing import Any, Dict, List
+from typing import Any, Callable, cast
 
 import msgspec
 import pytest
 from pydantic import SecretStr
+from pytest import FixtureRequest
 
 from litestar import MediaType, Response
 from litestar.contrib.pydantic import PydanticInitPlugin, _model_dump
 from litestar.exceptions import ImproperlyConfiguredException
 from litestar.serialization import get_serializer
-from tests import (
+from tests.models import (
+    DataclassPersonFactory,
     MsgSpecStructPerson,
-    PydanticDataClassPerson,
+    PydanticDataclassPerson,
     PydanticPerson,
-    PydanticPersonFactory,
-    VanillaDataClassPerson,
 )
 
-person = PydanticPersonFactory.build()
-secret = SecretStr("secret_text")
-pure_path = PurePath("/path/to/file")
-path = Path("/path/to/file")
+person = DataclassPersonFactory.build()
 
 
 class _TestEnum(enum.Enum):
     A = "alpha"
     B = "beta"
 
 
-@pytest.mark.parametrize("media_type", [MediaType.JSON, MediaType.MESSAGEPACK])
```

Comment:
This test is a bit more verbose now but I found the old one not very easy to follow and it's always a good idea to be more explicit in tests